staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_group.go

```diff
@@ -342,7 +344,10 @@ func (c *$.GroupGoName$$.Version$Client) run() {
 				}
 				clients[i] = client
 			}
+			c.mux.Lock()
+			$.klogInfo|raw$("Reset restClients. length %v -> %v", len(c.restClients), len(clients))
```

Comment:
BTW, you can try print out the host value to check config changes especially when # of clients decreases.

Reply:
that sounds good if the host info can help debug further  config related issues. this should not stress logging size with a simple stress here.

Reply:
actually after chatting with Ying. the config logging is already done in the config file. so we don't need to log here.